docs/guide/aws/cluster-manager.md

```diff
@@ -0,0 +1,45 @@
+## Cluster Manager
+
+Cluster Managers can manage multiple clusters across regions/data-centers and/or clouds. It is a global cluster manager which will run on Triton and manages Kubernetes environments. This cluster manager will manage environments running on any region of any supported cloud.
```

Comment:
The example shows the cluster-manager on AWS while the description above says that it runs on triton.